packages/sirius-web/backend/sirius-web/src/test/java/org/eclipse/sirius/web/application/services/EditingContextLifecycleTests.java

```diff
@@ -115,4 +138,34 @@ public void givenEditingContextProperlyLoadedWhenItIsModifiedAndPersistedThenThe
             assertThat(ePackage.getName()).isEqualTo("Sample Updated");
         }
     }
+
+    @Test
+    @GivenSiriusWebServer
+    @DisplayName("Given semantic data with a dependency, when the loading is performed, then the semantic data are available in the editing context")
+    public void givenSemanticDataWithDependencyWhenLoadingIsPerformedThenSemanticDataAvailableInEditingContext() {
+        Optional<Library> library = this.librarySearchService.findByNamespaceAndNameAndVersion("java", "Java Standard Library", "17.0.0");
+        assertThat(library).isPresent();
+        var result = this.semanticDataUpdateService.addDependency(null, PapayaIdentifiers.PAPAYA_EDITING_CONTEXT_ID, library.get().getSemanticData().getId());
```

Comment:
I manually added the dependency to setup my test. This could be simplified by moving the dependency in the initialization script (this would imply to move the Java Standard Library in the script too, but since it hasn't been done in the previous PR I didn't do it here).